source/devices/J7_Family/linux/Release_Specific_Release_Notes.rst

```diff
@@ -468,6 +444,15 @@ System firmware Known Issues
 Change Requests
 ===============
 
+SDK features descoped from 11.01 release
+----------------------------------------
+.. csv-table::
+  :header: "ID", "Head Line", "Platform", "Original Fix Version", "New Fix Version"
+  :widths: 20, 90, 90, 20, 20
+
+  JACINTOREQ-7915, "J7: Support for Kexec/Kdump in Linux", "All J7 Platforms", 11.01.00, 12.00.00
+  JACINTOREQ-7596, "[Linux]: Support for Headless display using QT", "J722S,J7ES,J784S4,J721S2", 11.01.00, "Future Release"
```

Comment:
https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace